bucket/kotlin-native.json

```diff
@@ -2,11 +2,12 @@
     "homepage": "https://kotlinlang.org/",
     "version": "0.9.3",
```

Comment:
Shouldn’t the version bump to 1.3.11 too?

Reply:
My bad... Missed this field